archetypes/catalog/catalog.xml.mustache

```diff
@@ -57,4 +57,10 @@
                summary="Helidon SE application that uses the dbclient API"
                description="Helidon SE application that uses the dbclient API with an in-memory H2 database"
                tags="se,rest,db" />
+    <archetype artifactId="helidon-oci-mp"
+               name="oci"
+               title="Helidon MP OCI"
+               summary="Helidon MP application that uses the OCI Integration"
+               description="Helidon MP application that uses the OCI Integration based on SDK"
```

Comment:
I see what you were trying to do on matching the verbiage from above, but honestly, I think the "based on SDK" part should be removed.

Reply:
Fixed